sdks/java/extensions/sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -160,7 +160,12 @@ SqlCreate SqlCreateTable(Span s, boolean replace) :
     <TABLE> ifNotExists = IfNotExistsOpt()
     id = CompoundIdentifier()
     fieldList = FieldListParens()
-    <TYPE> type = StringLiteral()
+    <TYPE>
```

Comment:
I can pull this out into a separate PR if you wish. We don't currently register `type` with the parser, but that isn't a problem because the parser performs no validation on `SimpleIdentifier`. This just enables us to use unquoted types. For example `TYPE text` instead of `TYPE 'text'`.

Reply:
let's remove it from this PR for more visibility of what's changed. IMO `TYPE text instead of TYPE 'text'` is a confused point, if it doesn't support `TYPE jdbc`, `TYPE kafka`, or `TYPE mytable`.

Reply:
Ok, dropping this and adding it to #5601